modules/apigee_edge_teams/apigee_edge_teams.links.action.yml

```diff
@@ -1,18 +1,18 @@
 apigee_edge_teams.team.add_members:
   route_name: entity.team.add_members
-  title: 'Add members'
+  title: 'Add team members'
```

Comment:
"Team" as string could not be hardcoded in these places, because the team entity could be aliased in the system, just like any other Apigee Edge, which means it could be called "Company", "Organization", "Pink fluffy unicorn", whatever else :) Which means this string should adopt the _current_ entity label of the team entity, ex.: `Add organization members`. Run a quick search on the code for `$this->team->label()`.

Reply:
Updated PR.